topiary/tests/samples/expected/ocamllex.mll

```diff
@@ -0,0 +1,67 @@
+(**************************************************************************)
+(*  Taken from https://github.com/colis-anr/morbig/                       *)
```

Comment:
If I recognize it correctly, this is actually from paris-branch/dancelor!

Reply:
Whoops! That's a TODO then. I should also take a look at the license and do it correctly.

Reply:
Dancelor has no license and therefore re-use is not permitted. It is mostly laziness from my end, but that's how it applies for now. However, I am the sole author of the file in question, so I will happily be giving you free use of it.